cantools/database/can/signal.py

```diff
@@ -191,8 +193,8 @@ def __init__(self,
                  length: int,
                  byte_order: ByteOrder = 'little_endian',
                  is_signed: bool = False,
-                 initial: Optional[int] = None,
-                 invalid: Optional[int] = None,
+                 raw_initial: Optional[int] = None,
+                 raw_invalid: Optional[int] = None,
```

Comment:
```suggestion
                 raw_initial: Optional[Union[int, float]] = None,
                 raw_invalid: Optional[Union[int, float]] = None,
```

(and possibly also `bytes`, but I think this is not supported by anything at the moment...)